wiki/Guides/Livestreaming_osu!/en.md

```diff
@@ -0,0 +1,65 @@
+
+![](img/stream-info.png "Editing your stream's information")
+
+---
+
+After editing your stream's information, click the `Done` button. Now click on the `Preferences` button on the left, then `Channel`. Look for the `Primary Stream key` field and click `Copy`. **Do not share this key**. For now, paste the copied stream key in a notepad.
+
+![](img/stream-key.png "Getting the stream key")
+
+## OBS Studio
+
+After you create and configure your Twitch account, the next thing you will need is a streaming app. This guide will use OBS Studio; however, you are free to check any other alternatives (e.g. [XSplit Broadcaster](https://www.xsplit.com/broadcaster "XSplit Broadcaster")).
+
+Proceed to the [OBS Studio's website](https://obsproject.com/ "Open Broadcaster Software") and download the installer for your operating system. Open the installer and follow the steps to install the app.
+
+### Configuring OBS
+
+When you launch OBS Studio for the first time, the auto-configuration wizard will open. Choose `Optimize for streaming, recording is secondary` and click `Next`.
+
+![](img/configuration-wizard.png "OBS Studio's auto-configuration wizard")
+
+The next section to be configured are video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.
+
+Finally, you will need to configure the stream information for OBS Studio to livestream on Twitch. Click the `Use Stream Key` button, and enter the Primary Stream key that was copied earlier into the field. Leave other options checked as they are, and click `Next`.
+
+![](img/adding-stream-key.png "Configuring stream information in OBS")
+
+Now OBS Studio will perform auto-configuration, in order to find the best settings for your device. Press `Apply Settings` after the process is finished.
+
+### Adding a scene
+
+The last thing you will need to start streaming is a scene containing the osu! window. To create one, right-click on the `Scenes` box from OBS' main window and select `Add` to be prompted with a scene creation window. Name it "osu!", and click `OK` to continue.
+
+Now we will add a source for OBS to capture game's window. Before we continue, open osu! to make it possible. Right-click on the `Sources` box, and select `Add` -> `Game Capture` to be prompted with a source selection window. Name the source whatever you want it to be called ("osu!" or "Game window" are fine), and click `OK`. Select `Capture specific window` from the `Mode` dropdown, and select `[osu!.exe]: osu!` for the `Window` list. Then, check `Capture Cursor` if it's not already, and click `OK`).
```

Comment:
```suggestion
Now OBS Studio needs a source to capture osu!. First, open osu!, then—in OBS Studio—right-click on the `Sources` box and select `Add` -> `Game Capture`. This will open the source selection dialog. You can name the source as "osu!", but it can be whatever you want, then click `OK`. Select `Capture specific window` from the `Mode` dropdown then select `[osu!.exe]: osu!` from the `Window` list. Then, check `Capture Cursor` if it's not already, and click `OK`.
```
if Capture Cursor is checked, wouldn't this make the hardware cursor visible? (This suggestion does not change the last sentence to address this)

Reply:
it doesnt make the hardware cursor visible, but it looks like unchecking this option doesnt change anything anyways, so i will remove it completely ¯\_(ツ)_/¯